physionet-django/project/test_views.py

```diff
@@ -1150,7 +1150,8 @@ def setUpTestData(cls):
         cls.invalid_size_data_3 = {'filename': 'random.txt'}
         cls.invalid_filename_data_1 = {'size': 250000, 'filename': 'ran dom.txt'}
         cls.invalid_filename_data_2 = {'size': 250000, 'filename': 'random§.txt'}
-        cls.valid_data = {'size': 250000, 'filename': 'random.txt'}
+        cls.invalid_filename_length_data_2 = {'size': 250000, 'filename': 'invalid' * 100 + '.txt'}
```

Comment:
Should probably be called `...data_1`, not `...data_2`.

Reply:
oh sorry. yes. that is correct. fixing this